### PR TITLE
Fix/delft3dgt 871 overal styling

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: src/components/ConfirmDialog.vue [hunk 1]
```diff
@@ -42,6 +42,7 @@ export default {
     }
   },
   mounted () {
+    console.log('mounted')
```
Comment (SiggyF) [inline src/components/ConfirmDialog.vue]: remove

File: src/components/ModelControlMenu.vue [hunk 2]
```diff
@@ -62,11 +62,11 @@
         </a>
       </li>
 
-      <li role="separator" class="divider"></li>
+      <li role="separator" class="dropdown-divider"></li>
 
       <li :class="{ disabled: numSelectedModels === 0 || (option.onlyFinished && !someSelectedModelsAreFinished()) }" v-for="(option, key) in downloadOptions" :key="key">
         <div class="dropdown-menu-checkbox" @click.self.stop.prevent="toggle(key)">
-          <input type="checkbox" class="downloadoption"  :value="key" v-model="downloadOptions['key']" :disabled="numSelectedModels === 0 || (option.onlyFinished && !someSelectedModelsAreFinished())"> {{ option.verbose }}
+          <input type="checkbox" class="downloadoption"  :value="downloadOptions[key].active" v-model="downloadOptions[key].active" :disabled="numSelectedModels === 0 || (option.onlyFinished && !someSelectedModelsAreFinished())"> {{ option.verbose }}
```
Comment (SiggyF) [inline src/components/ModelControlMenu.vue]: `key`  or `'key'`

File: src/components/ModelControlMenu.vue [hunk 3]
```diff
@@ -79,7 +79,7 @@
     </ul>
 
         <!-- Confirm dialog for control checks -->
-        <confirm-dialog v-if="updateModelsBy.name" :confirm-button-title="capitalizeFirst(updateModelsBy.name)" :dialogId="`confirm-${updateModelsBy.name}`" @confirm="confirm">
+        <confirm-dialog v-if="updateModelsBy.name" :confirm-button-title="capitalizeFirst(updateModelsBy.name)" :dialogId="`confirm-${updateModelsBy.name}`" @confirm="confirm" @cancel="updateModelsBy = {}">
```
Comment (SiggyF) [inline src/components/ModelControlMenu.vue]: cancel method?

File: src/views/ModelDetails.vue [hunk 4]
```diff
@@ -546,7 +546,7 @@ export default {
       window.open(`api/v1/scenes/${id}/export/?format=json&${downloadOptions.join('&')}`)
     },
     hasPostProcessData () {
-      return (Object.keys(_.get(this.activeModel, 'data.info.postprocess_output', {})).length > 0)
+      return (Object.keys(_.get(this.activeModel, 'data.info.postprocess_output.files', {})).length > 0)
```
Comment (SiggyF) [inline src/views/ModelDetails.vue:550]: Any comments?

**Review by SiggyF — APPROVED** [review]
Looks good.

File: src/components/ModelControlMenu.vue (hunk 2 above)
Comment (CindyvdVries) [reply src/components/ModelControlMenu.vue]: key

**Pull request merged**